virtcontainers/qemu.go

```diff
@@ -621,16 +627,24 @@ func (q *qemu) startSandbox(timeout int) error {
 		timeStart := time.Now()
 		go func() {
 			scanner := bufio.NewScanner(stderr)
+			var sent bool
 			for scanner.Scan() {
-				if strings.Contains(scanner.Text(), "Waiting for vhost-user socket connection...") {
+				if q.config.Debug {
+					q.Logger().WithField("source", "virtiofsd").Debug(scanner.Text())
+				}
+				if !sent && strings.Contains(scanner.Text(), "Waiting for QEMU socket connection...") {
 					sockReady <- nil
-					return
+					sent = true
```

Comment:
I'm not sure it's useful to keep running the loop after sockReady <- nil since the kata-runtime process will terminate and this goroutine will disappear along with it.  Continuing to log from a goroutine makes sense in the v2 shim model, but I don't think it works in the classic kata-runtime OCI runtime model.

Reply:
Yes, indeed. It can only be a best effort in the OCI command line case. And it works better for shimv2.

Reply:
Looks good.